cli/tools/jupyter/mod.rs

```diff
@@ -71,13 +80,25 @@ pub async fn kernel(
         connection_filepath
       )
     })?;
-
+  let (test_event_sender, test_event_receiver) =
+    unbounded_channel::<TestEvent>();
+  let test_event_sender = TestEventSender::new(test_event_sender);
+  let stdout = StdioPipe::File(test_event_sender.stdout());
+  let stderr = StdioPipe::File(test_event_sender.stderr());
   let mut worker = worker_factory
     .create_custom_worker(
       main_module.clone(),
       permissions,
-      vec![ops::jupyter::deno_jupyter::init_ops(stdio_tx)],
-      Default::default(),
+      vec![
+        ops::jupyter::deno_jupyter::init_ops(stdio_tx.clone()),
+        crate::ops::testing::deno_test::init_ops(test_event_sender.clone()),
+      ],
+      // FIXME(nayeemrmn): Test output capturing currently doesn't work.
```

Comment:
Hm, shouldn't we be able to capture the test output the same way `deno test` does it? Isn't it a matter of forwarding it to a proper channel?

Reply:
It causes all logged output in the repl to be buffered and then flushed to stdout at once when a test event is processed.